tests/llmcompressor/transformers/finetune/data/test_registry.py

```diff
@@ -57,3 +58,19 @@ def test_open_platypus_initializes(tiny_llama_tokenizer):
     assert op_manager.text_column == "text"
     assert not op_manager.padding
     assert op_manager.max_seq_length == data_args.max_seq_length
+
+
+@pytest.mark.usefixtures("tiny_llama_tokenizer")
+def test_pile_eval_initializes(tiny_llama_tokenizer):
```

Comment:
Could you also add a test for loading and tokenizing a small subset to `test_dataset_loading.py`?